openedx/core/djangoapps/credentials/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
Also this is just a thin wrapper not an API Client. This wrapper will further use edx-rest-api client to communicate with future credentials IDA.